{"name": "test_single_resource_not_found", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n  +404\n  -200", "trace": "self = <test_requests.TestGetRequests object at 0x7fcabbddca30>\n\n    def test_single_resource_not_found(self):\n        get_req = requests.get('https://reqres.in/api/unknown/23')\n        get_req_json = get_req.json()\n>       assert get_req.status_code == 200\nE       assert 404 == 200\nE         +404\nE         -200\n\ntest_requests.py:39: AssertionError"}, "start": 1612048412364, "stop": 1612048412762, "uuid": "0f8a488a-86ce-442b-a4f8-b2c5c0c942ff", "historyId": "809cba64e21dbb5375ced0b5a8ed5640", "testCaseId": "18d9767c9d6e57814e1c5bd0c27061b3", "fullName": "api_testing.homework3.test_requests.TestGetRequests#test_single_resource_not_found", "labels": [{"name": "parentSuite", "value": "api_testing.homework3"}, {"name": "suite", "value": "test_requests"}, {"name": "subSuite", "value": "TestGetRequests"}, {"name": "host", "value": "kali"}, {"name": "thread", "value": "2656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "api_testing.homework3.test_requests"}]}